Euler1PS.js

```diff
@@ -0,0 +1,13 @@
+let sum = 0;
+
+function findTheSum(max) {
+    for (let i = 3; i < max; i++) {
+        if (i % 3 === 0 || i % 5 === 0) {
```

Comment:
Do you want object equality (`===`) or numeric equality (`==`)? It turns out to be the same thing in most Javascript interpreters, but that's by accident. Also, Javascript is one of the few languages where that difference matters.

Reply:
So there was no type coercion I did object equality, although with this file being so small I don't believe it would have mattered